proxy/envoy/ingress.go

```diff
@@ -372,3 +387,20 @@ func extractPortAndTags(dst *config.DestinationWeight) (*model.Port, model.Tags,
 
 	return port, tags, nil
 }
+
+func writeIstioAuthSecret(secret *model.IstioAuthSecret, mesh *config.ProxyMeshConfig) error {
```

Comment:
Looks like we are manually fetch and write the secrets to dedicated path, just like Proxy, right?